README.md

```diff
@@ -277,6 +277,14 @@ sequencer.insertSteps(index,module_name,optional_options);
 
 return value: **`sequencer`** (To allow method chaining)
 
+###Importing an independent module
+The `loadNewModule` method can be used to import a new module inside sequencer. Modules can be downloaded via npm, yarn or cdn and are imported with a custom name
+
+```js
+const module = require('sequencer-moduleName')
```

Comment:
We should note if this is not usable in the browser.

Reply:
and how to do it in a different way if you are in the browser, or what alternatives are?

Reply:
So maybe a comment on this line about the `require()` usage?

Reply:
so adding `If you wish to load a new module at runtime, it will need to avoid using `require()` -- unless it is compiled with a system like browserify or webpack.`